docs/source/tutorial/chapter1.rst

```diff
@@ -0,0 +1,51 @@
+Chapter 1
+=========
+
+First Fortran MUSICA Program
+----------------------------
+The MUSICA library can be used within a fortran program.
+To get started, let us create a simple program that links
+to MUSICA and prints the version of MICM.
+
+Here are the contents of the program `demo.f90`:
+
+.. code-block:: f90
+
+  program demo
+      use musica_util, only: string_t
+      use musica_micm, only: get_micm_version
+      implicit none
+      type(string_t) :: micm_version
+      micm_version = get_micm_version()
+      print *, "MICM version ", micm_version%get_char_array()
+  end program demo
```

Comment:
I would suggest that we move all the tutorial code snippets to separate source code files that get compiled and run as tests. This way we can ensure that if we break a part of the tutorial with updates to the MUSICA source code, a test will fail.

We do something similar in MICM:

https://github.com/NCAR/micm/blob/ee5c03e904d9dc3d60619dc9dbce06232bd32852/docs/source/user_guide/rate_constant_tutorial.rst?plain=1#L46-L49

Reply:
Yes, let me do that ...